### ISSUE TITLE
Support of Complex identifiers

### ISSUE THREAD

**Issue by egonw** [wikipathways/GPML2RDF#75]
During the #covidpathways project, we worked on one solution to support complex identifiers. Ideally, the `Complex` groups have identifiers themselves, but until then, this my be a solution. See this patch: https://github.com/wikipathways/GPML2RDF/commit/193d015c18d72d812a51865d17a6787d39a69dfe

THIS IS AN ONGOING DISCUSSION.

**Comment by egonw** [comment]
cc @DeniseSl22

**Comment by DeniseSl22** [comment]
This would also need to be implemented in PV + GPML, I suppose?

**Comment by egonw** [comment]
Yes, but it currently is not. The trick used right now is as follows: each GPML Complex with (exactly) one DataNode of @Type="Complex" with an identifier inherits the identifiers from that DataNode.

**Comment by DeniseSl22** [comment]
Okay, that sounds cleaver.... updating the GPML to accommodate for this is much more work indeed (but that's probably for the future, because I think we want to build complexes with more then 1 DataNode ;) ).
